queries.yaml

```diff
@@ -338,6 +339,15 @@ layers:
       - vectordatasource.transform.add_id_to_properties
       - vectordatasource.transform.remove_feature_id
     area-inclusion-threshold: 1
+  # optional inline layer, if the file is found on disk it will be used
+  landmarks:
+    geometry_types: [Point, Polygon, MultiPolygon]
+    clip: false
+    area-inclusion-threshold: 1
+    simplify_before_intersect: false
+    simplify_start: 0,
+    layer_path: ./metatile-layers/landmarks.geojson
```

Comment:
this property signifies an "inline" layer. when the features are loaded from file we expect them to be already processed. that is we will not try to load a `landmarks.yaml` and do processing on the data that is in the geojson file. we expect the tags and values to be there and the coordinates to already be in mercator projection

Reply:
Could we make the name more explicit about its behavior, like `pre_processed_layer_path` might be a better name?